scripts/install.sh

```diff
@@ -305,8 +352,20 @@ if [[ $namespace_scope_mode == "true" ]]; then
   rm crd/kustomization.yaml.bak
 fi
 
-info "Waiting for ModelMesh Serving controller pod to be up ..."
-wait_for_pods_ready "-l control-plane=modelmesh-controller"
+
+if [[ ! -z $modelmesh_serving_image ]]; then 
+  cp manager/kustomization.yaml.ori  manager/kustomization.yaml
+  rm manager/kustomization.yaml.ori 
+fi
+
+if [[ $enable_self_signed_ca == "true" ]]; then
+  cp certmanager/kustomization.yaml.ori  certmanager/kustomization.yaml
+  cp default/kustomization.yaml.ori  default/kustomization.yaml
+  rm certmanager/kustomization.yaml.ori default/kustomization.yaml.ori
+else
+  info "Waiting for ModelMesh Serving controller pod to be up..."
+  wait_for_pods_ready "-l control-plane=modelmesh-controller"
+fi
```

Comment:
It looks like this should work: combining the `if [[ $enable_self_signed_ca == "true" ]];` block below into this one and only have one place to call `wait_for_pods_ready`
```suggestion
if [[ $enable_self_signed_ca == "true" ]]; then
  cp certmanager/kustomization.yaml.ori  certmanager/kustomization.yaml
  cp default/kustomization.yaml.ori  default/kustomization.yaml
  rm certmanager/kustomization.yaml.ori default/kustomization.yaml.ori
  
  info "Enabled Self Signed CA: Generate certificates and restart controller"
  # Delete dummy secret for webhook server
  kubectl delete secret modelmesh-webhook-server-cert 

  ../scripts/self-signed-ca.sh --namespace $namespace
fi

info "Waiting for ModelMesh Serving controller pod to be up..."
wait_for_pods_ready "-l control-plane=modelmesh-controller"
```

Reply:
You are right. I updated it